Maple2.File.Parser/Enum/WeddingRewardType.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Maple2.File.Parser.Enum;
```

Comment:
**suggestion:** Add documentation for enum values

Consider adding XML comments to explain what each enum value represents, especially for less obvious ones like 'logoutPanelty'. This will improve code readability and maintainability.